client/src/js/utils/sagas.js

```diff
@@ -3,10 +3,11 @@
  *
  * @module sagaUtils
  */
-import { push } from "connected-react-router";
+import { replace } from "connected-react-router";
 import { get, includes } from "lodash-es";
 import { matchPath } from "react-router-dom";
 import { all, put } from "redux-saga/effects";
+import { pushState } from "../app/actions";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['pushState' is defined but never used.](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=4938342)